.github/workflows/docs.yml

```diff
@@ -0,0 +1,23 @@
+name: docs
+
+on: [push, pull_request]
+
+jobs:
+  docs:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
could we also use python 3.9 here?

Reply:
We could, but I am a bit conservative when it comes to Python version upticks. I would prefer to keep 3.9 as supported but use the previous stable 3.8 as the „main” version, used for building, security, docs etc.

Reply:
I have updated my stance and the code. 😄